src/jdk.unsupported/share/classes/com/sun/nio/file/SensitivityWatchEventModifier.java

```diff
@@ -32,9 +32,18 @@
  * Defines the <em>sensitivity levels</em> when registering objects with a
  * watch service implementation that polls the file system.
  *
+ * @deprecated
+ * The polling implementation of {@link java.nio.file.WatchService WatchService}
+ * no longer recognizes this {@link java.nio.file.WatchEvent.Modifier
+ * WatchEvent.Modifier}.  This was the only {@code WatchService} in the JDK
+ * which used this {@code WatchEvent.Modifier}.  Therefore this
+ * {@code WatchEvent.Modifier} class is vestigial and should be removed in a
+ * future release.
```

Comment:
"no longer recognizes" suggests that using them will lead to an error but it's actually that they will be just ignored. 

Also "should be removed in a future release" will duplicate text generated by javadoc. The JDK docs build doesn't generate javadoc for the jdk.unsupported module so maybe it doesn't matter but here's a suggestion for something a bit briefer:

"The sensitivity levels were historically used by polling based WatchService implementations to configure the polling interval. They are are no longer used. The WatchService implementations in the JDK ignore these WatchEvent modifiers if they are specified when registering a directory to be watched."

Reply:
Thanks, that is much better; used in ca92fdba0897b7acfedc3bc933b874c3b421a0a3.